common/source/docs/common-rtf.rst

```diff
@@ -90,19 +73,26 @@ Subs from Partners
 
 * `Blue Robotics - BlueROV2 <https://bluerobotics.com/store/rov/bluerov2/>`__
 
-
 Vehicles from Non-Partners
 ==========================
 
+* `AION ROBOTICS - R6 UGV <https://www.aionrobotics.com/r6>`__
```

Comment:
```suggestion
* `AION ROBOTICS <https://www.aionrobotics.com>`__
```
no direct product links on the site...maybe should remove entirely

Reply:
yup, good idea, removed.